utbot-framework/src/main/kotlin/org/utbot/framework/codegen/tree/CgMethodConstructor.kt

```diff
@@ -1932,6 +1933,7 @@ open class CgMethodConstructor(val context: CgContext) : CgContextOwner by conte
             // Exceptions and annotations assignment must run after the statements block is build,
             // because we collect info about exceptions and required annotations while building the statements
             exceptions += collectedExceptions
+            // TODO: can we get annotations from scope? This will allow to use `addAnnotation` without return type.
             annotations += testFrameworkManager.createDataProviderAnnotations(dataProviderMethodName)
```

Comment:
Does that not work as expected here?
```suggestion
            testFrameworkManager.createDataProviderAnnotations(dataProviderMethodName)
            annotations += collectedMethodAnnotations 
```

Reply:
Also, I think `createDataProviderAnnotations` should be renamed to `addDataProviderAnnotations` since it has a side effect of modifying `collectedMethodAnnotations`.